src/databricks/labs/ucx/hive_metastore/grants.py

```diff
@@ -47,6 +50,7 @@ class Grant:
     table: str | None = None
     view: str | None = None
     udf: str | None = None
+    external_location: str | None = None
```

Comment:
There's no such thing as external location in legacy table acl and hive metastore

Reply:
Removed this since we are just granting access to database and table